src/modules/foodsUI.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable */
+import getMealInfo from "./getMeals.js";
+import { getLikes } from "./userinvolve.js";
+
+import mealsCounter from './mealCount.js';
+
+const foodItemsDiv = document.getElementById('food-items');
+
+// event listeners
+foodItemsDiv.addEventListener('click', getMealInfo);
+
+const foodsUI = async () => {
+  fetch('https://www.themealdb.com/api/json/v1/1/search.php?f=g')
+    .then((res) => res.json())
+    .then((data) => {
+      getLikes().then((datalikes) => {
```

Comment:
- [ ] Great job, however you're still using promises (then) within your async here. I kindly suggest you use async and await.

## EXAMPLE
```
  async function getMeals() {
    const response = await fetch('https://www.themealdb.com/api/json/v1/1/search.php?s=');
    const data = await response.json();
    const meals = data.meals;
    return meals;
  }
```